### PR TITLE
Implement exercise food-chain

### LINKED ISSUE #750: food-chain: implement exercise
There are exercises that are not yet implemented in Python track (the full list of them - https://github.com/exercism/python/issues/417#issuecomment-334841778).
You can help us change this situation by implementing this one. Ref #417

The guide ["Implement an exercise from specification"](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md) will help you succeed.

If you still have questions, I encourage you to ask them before starting working on the issue.

I also suggest claiming the issue by creating an empty PR as stated [here](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md#avoiding-duplicate-work)

Please note that there are some special ['closing issue' keywords](https://help.github.com/articles/closing-issues-using-keywords/) that you can put in description of your PR and Github will take care of closing the corresponding issue automatically after merge.
This simple thing helps us a lot, and makes our life a bit easier

### PR DESCRIPTION
**Pull request by LU4XLII**
-----------
Fixes #750

### CODE REVIEW

**Comment by m-a-ge** [comment]
@LU4XLII could you please fix code style issues:
```text
2.78s$ flake8
./exercises/food-chain/example.py:33:80: E501 line too long (80 > 79 characters)
./exercises/food-chain/food_chain_test.py:15:80: E501 line too long (92 > 79 characters)
./exercises/food-chain/food_chain_test.py:22:80: E501 line too long (92 > 79 characters)
./exercises/food-chain/food_chain_test.py:30:80: E501 line too long (92 > 79 characters)
./exercises/food-chain/food_chain_test.py:39:80: E501 line too long (92 > 79 characters)
./exercises/food-chain/food_chain_test.py:49:80: E501 line too long (92 > 79 characters)
./exercises/food-chain/food_chain_test.py:58:1: E302 expected 2 blank lines, found 1
./exercises/food-chain/food_chain_test.py:61:80: E501 line too long (82 > 79 characters)
./exercises/food-chain/food_chain_test.py:63:1: E305 expected 2 blank lines after class or function definition, found 1
The command "flake8" failed and exited with 1 during .
```

**Comment by LU4XLII** [comment]
Fixed up

File: exercises/food-chain/food_chain_test.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+I don't know why she swallowed the fly. Perhaps she'll die.
+
+I know an old lady who swallowed a horse.
+She's dead, of course!"""
+
+song = song.replace("\n", "").replace(" ", "").lower()
+
+
+class FoodChainTest(unittest.TestCase):
+
```
Comment (m-a-ge) [inline exercises/food-chain/food_chain_test.py]: Please implement other tests from https://github.com/exercism/problem-specifications/blob/master/exercises/food-chain/canonical-data.json

Could you also leave a comment stating what version of `canonical-data.json` the tests were adopted as discussed in #784 if applicable?
The format is:
```text
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

Comment (LU4XLII) [reply exercises/food-chain/food_chain_test.py]: Ok I'll work on this now.

**Comment by LU4XLII** [comment]
All tests implemented

**Comment by m-a-ge** [comment]
Thanks for working on this, @LU4XLII 🏆

**Pull request merged**